packages/i18n/src/fr.ts

```diff
@@ -0,0 +1,66 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'fr',
+  schema:           (issue) => `Typage invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
+  specific: {
+    bic:            (issue) => `BIC invalide: reçu ${issue.received}`,
+    bytes:          (issue) => `Octet invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
+    creditCard:     (issue) => `Carte de crédit invalide: reçu ${issue.received}`,
+    cuid2:          (issue) => `Cuid2 invalide: reçu ${issue.received}`,
+    custom:         (issue) => `Valeur invalide: reçu ${issue.received}`,
+    customAsync:    (issue) => `Valeur invalide: reçu ${issue.received}`,
+    decimal:        (issue) => `Décimale invalide: reçu ${issue.received}`,
+    email:          (issue) => `Email invalide: reçu ${issue.received}`,
+    emoji:          (issue) => `Émoji invalide: reçu ${issue.received}`,
+    endsWith:       (issue) => `Fin invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
+    equal:          (issue) => `Valeur invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
+    excludes:       (issue) => `Contenu invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
+    finite:         (issue) => `Entier invalide: reçu ${issue.received}`,
+    hash:           (issue) => `Hash invalide: reçu ${issue.received}`,
+    hexColor:       (issue) => `Couleur hexa invalide: reçu ${issue.received}`,
+    hexadecimal:    (issue) => `Hexadécimal invalide: reçu ${issue.received}`,
+    imei:           (issue) => `IMEI invalide: reçu ${issue.received}`,
+    includes:       (issue) => `Contenu invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
+    integer:        (issue) => `Entier invalide: reçu ${issue.received}`,
+    ip:             (issue) => `IP invalide: reçu ${issue.received}`,
+    ipv4:           (issue) => `IPv4 invalide: reçu ${issue.received}`,
+    ipv6:           (issue) => `IPv6 invalide: reçu ${issue.received}`,
+    isoDate:        (issue) => `Date invalide: reçu ${issue.received}`,
+    isoDateTime:    (issue) => `Date et heure invalide: reçu ${issue.received}`,
+    isoTime:        (issue) => `Temps invalide: reçu ${issue.received}`,
+    isoTimeSecond:  (issue) => `Temps invalide (secondes): reçu ${issue.received}`,
+    isoTimestamp:   (issue) => `Timestamp invalide: reçu ${issue.received}`,
+    isoWeek:        (issue) => `Semaine invalide: reçu ${issue.received}`,
+    length:         (issue) => `Longeur invalide: attendu ${issue.expected} mais reçois ${issue.received}`,
```

Comment:
- En français, [une virgule précède toujours le mot «&thinsp;mais.&thinsp;»](https://vitrinelinguistique.oqlf.gouv.qc.ca/23445/la-ponctuation/virgule/la-virgule-avant-et-apres-mais);
- Les temps est toujours inconsistent «&thinsp;attendu&thinsp;» / «&thinsp;reçois&thinsp;».

```suggestion
    length:         (issue) => `Longeur invalide: attendu ${issue.expected}, mais reçu ${issue.received}`,
```

Reply:
En France, la virgule n'est pas obligatoire (peut-être au Québec si de ce que je comprends.) Idem pour les temps, cela n'est pas bloquant, mais je vais faire les modifs

Reply:
Tout juste, elle n’est effectivement [pas requise devant une conjonction de coordination](https://dictionnaire.lerobert.com/guide/conjonctions-de-coordination). Toutes mes excuses 🙇 – je pensais qu’il s’agissait d’une règle universelle.